src/card.css

```diff
@@ -0,0 +1,96 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  padding: 16px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__image {
+  align-self: center;
+
+  height: 134px;
+  margin-top: 16px;
+}
+
+.product-card__header {
+  margin: 40px 0 0;
+
+  color: var(--main-accent);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.product-card__code {
+  margin-top: 4px;
+
+  color: var(--secondary);
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.product-card__rating-section {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-top: 16px;
+}
+
+.product-card__reviews {
+  color: var(--main-accent);
+  font-size: 10px;
```

Comment:
Isn't it a default color and size?

Reply:
Neither design nor requirements don't set obvious default font size or font color